mmv1/products/privilegedaccessmanager/Entitlement.yaml

```diff
@@ -0,0 +1,230 @@
+                  required: true
+  - !ruby/object:Api::Type::String
+    name: maxRequestDuration
+    description: |
+      The maximum amount of time for which access would be granted for a request.
+      A requester can choose to ask for access for less than this duration butnever more
+      Format: calculate the time in seconds and concatenate it with 's' i.e. 2 hours = "7200s", 45 minutes = "2700s"
+    required: true
+  - !ruby/object:Api::Type::String
+    name: state
+    description: Output only. The current state of the Entitlement.
+    output: true
+  - !ruby/object:Api::Type::Fingerprint
+    name: etag
+    description: |
+      For Resource freshness validation (https://google.aip.dev/154)
+    output: true
+  - !ruby/object:Api::Type::NestedObject
+    name: requesterJustificationConfig
+    description: |
+      Defines the ways in which a requester should provide the justification while requesting for access.
+    required: true
+    allow_empty_object: true
+    send_empty_value: true
+    properties:
+      - !ruby/object:Api::Type::NestedObject
+        name: notMandatory
+        description: |
+          The justification is not mandatory but can be provided in any of the supported formats.
+        properties: []
+        allow_empty_object: true
+        send_empty_value: true
+        conflicts:
+          - unstructured
+      - !ruby/object:Api::Type::NestedObject
+        name: unstructured
+        description: |
+          The requester has to provide a justification in the form of free flowing text.
+        properties: []
+        allow_empty_object: true
+        send_empty_value: true
+        conflicts:
+          - not_mandatory
+  - !ruby/object:Api::Type::NestedObject
+    name: additionalNotificationTargets
+    description: |
+      AdditionalNotificationTargets includes email addresses to be notified.
+    properties:
+      - !ruby/object:Api::Type::Array
+        name: adminEmailRecipients
+        description: |
+          Optional. Additional email addresses to be notified when a principal(requester) is granted access.
+        item_type: Api::Type::String
+      - !ruby/object:Api::Type::Array
+        name: requesterEmailRecipients
+        item_type: Api::Type::String
+        description: |
+          Optional. Additional email address to be notified about an eligible entitlement.
+parameters:
+  - !ruby/object:Api::Type::String
+    name: location
+    description: "TODO"
+    url_param_only: true
+    required: true
+    immutable: true
+  - !ruby/object:Api::Type::String
+    name: entitlementId
+    description: "TODO"
+    url_param_only: true
+    required: true
+    immutable: true
+  - !ruby/object:Api::Type::String
+    name: parent
+    immutable: true
+    required: true
+    url_param_only: true
+    description: |
+      Format: project/{project_id} or organization/{organization_number} or folder/{folder_number}
```

Comment:
In our API parent contains the location too,
so "project/{project_id}/locations/{region}" and so on